BDArmory/Weapons/Missiles/MissileLauncher.cs

```diff
@@ -1748,6 +1778,12 @@ IEnumerator BoostRoutine()
                     }
 
                 //thrust
+                if (useFuel && burnRate > 0 && ordinanceMass < boosterMass)
+                {
+                    ordinanceMass -= burnRate;
+                    part.UpdateMass();
+                    part.Update();
+                }
```

Comment:
You've missed the point of my previous comment. 
``part.UpdateMass()`` and ``part.update()`` are superfluous and do not need to be here,

Reply:
Sorry, I'm sorry, I misunderstood, fixing that